user-storage-jpa-legacy/src/main/resources/META-INF/services/org.keycloak.storage.UserStorageProviderFactory

```diff
@@ -0,0 +1 @@
+org.keycloak.quickstart.storage.user.MyExampleUserStorageProviderFactory
```

Comment:
@pedroigor - this should IMHO be EjbExampleUserStorageProviderFactory; this makes the integration test of the old legacy test fail.

Reply:
Yeah, the renaming went to far.